views/hole-ng.html

```diff
@@ -117,6 +117,9 @@ <h1>{{ .Data.Hole.Name }}</h1>
         column of the first result set will be printed, NULL values will be
         skipped, and the dialect is <a href=//sqlite.org/index.html>SQLite</a>.
     </div>
+    <div class="info viml">
+        Arguments are available on sequential lines in the current file.
```

Comment:
Might be worth mentioning in these info blocks that you need to write and close the file at the end of your script.